content/blog/2018-07-05-mchtoolbox.md

```diff
@@ -0,0 +1,78 @@
+
+### Our motivation
+
+We aimed to build a package that addresses a gap in the workflow of researchers working in maternal and child health: Reference data from representative populations is used in maternal and child health research and public health surveillance to calculate percentiles, z-scores, and related values. This is important for measures that naturally change over the lifespan, such as body mass index and blood pressure. Because code to calculate these quantities - if it exists! - may be only available in SAS (which is not freely available to all researchers), or is scattered among a few disconnected R packages, it can be difficult to calculate these variables and incorporate them into an R-centric workflow. We wanted to create something that would close this gap. As we formed project teams for the unconf, we were convinced that this project would be something we could create within the scope of the unconf that would tangibly benefit others.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I have been wanting someone to make a package that does this for years.  I had CDC code in SAS and always had to transfer my data. Thanks!</p>&mdash; Kate Kelsey (@CapNScurvy) <a href="https://twitter.com/CapNScurvy/status/999274443707891719?ref_src=twsrc%5Etfw">May 23, 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+### Using runconf to learn how to collaborate
+
+If you’re an [isolated data scientist](https://peerj.com/preprints/3160/) or if you tend to develop analyses on your own, collaboration usually consists of documenting your code and using Git/GitHub so future you knows what past you was up to. Using version control on your own is a great place to begin if you haven’t tried it before, and is fairly straightforward. However, we rapidly discovered that working collaboratively on GitHub  can get quite complicated when you have 5 people sitting around a table contributing to a code base in real time, and some changes in git workflow habits may be required! Thanks to the community and environment rOpenSci intentionally creates at the unconf, we had a short crash course from Jenny Bryan in [happy git with R](http://happygitwithr.com/) and the `usethis` package, and help from Maȅlle Salmon on setting up continuous integration, giving us a solid workflow for software development as we move forward. It can be a pretty big leap stepping into a highly collaborative project for the first time -- expect there to be challenges and absolutely don’t be afraid to ask for help! 
```

Comment:
s/Maȅlle/Maëlle/

On my machine, if I hold down the `e` key, I get options to select `ë`. If you can't fix this by the time I publish, I'm happy to do it.